### ISSUE TITLE
`--root-override` missing from usage output

### ISSUE THREAD

**Issue by gsingh93** [radxa-repo/rbuild#13]
The output of `./rbuild --help` doesn't show the `--root-override` flag. I was confused as I was trying to use `--native-build` which requires `sudo`, but the script doesn't let you run with `sudo` by default. I figured out this command existed by reading the source, but it should be added to the usage output.

**Comment by RadxaYuntian** [comment]
In general running everything as root is bad practice. For `rbuild` specifically it caused file permission issues that makes the development not as smooth. I had to [diagnose one of my colleague's system](https://github.com/radxa-repo/rbuild/commit/27344530599a5b4ef9a51e698fd19365d1b69a37) and the problem was indeed caused by running the script with `sudo`. So it is banned, and we provided [container runtime setup guide](https://radxa-repo.github.io/rbuild/first_image.html) to allow execution of the script without `sudo`.

However, building as root was necessary to for our GitHub workflow, and I think the usage is beyond `--native-build`. Since I was already bitten by sudo-happy people, I keep this flag internal to avoid more unnecessary support issues.

I'll double check if this flag can be merged with `--native-build`.

**Comment by gsingh93** [comment]
Thanks! I also prefer to not need to use `sudo`, but in this case it was necessary to get around this error:
```
2023/04/21 22:46:10 fakemachine not supported, running on the host!
2023/04/21 22:46:10 Action `Set up on-disk layout` failed at stage PreNoMachine, error: Failed to setup loop device
```

Do you have another suggestion for how to deal with this without `sudo`?

**Comment by RadxaYuntian** [comment]
From that error message you actually have to use with `sudo`. I'll merge those 2 options in one later.

**Comment by RadxaYuntian** [comment]
Fixed in a695c01353140d32fe2ecb6d59a15792a5301f0e.
